src/jemalloc.c

```diff
@@ -2201,6 +2201,9 @@ static bool
 malloc_init_hard(void) {
 	tsd_t *tsd;
 
+	if (config_limit_usize_gap) {
+		assert(TCACHE_MAXCLASS_LIMIT <= USIZE_GROW_SLOW_THRESHOLD);
```

Comment:
nit: for the `if (A) { assert(B) }` logic, we usually do `assert(!A || B)`

Reply:
There are more assertions added here in the following commits, I think using an `if` to guard them all may be neater, WDYT?